bin/check-rabbitmq-consumers.rb

```diff
@@ -44,9 +45,9 @@ class CheckRabbitMQConsumers < Sensu::Plugin::Check::CLI
          boolean: true,
          default: false
 
-  option :user,
+  option :username,
```

Comment:
I like making it consistent but this makes it a breaking change. This will need to be called out in the CHANGELOG.